src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/TheSystemsNodes/GriperNode.java

```diff
@@ -0,0 +1,9 @@
+package edu.greenblitz.tobyDetermined.Nodesssss.TheSystemsNodes;
+
+import edu.greenblitz.tobyDetermined.Nodesssss.GBNode;
+
+public class GriperNode extends GBNode {
```

Comment:
Also, could you elaborate on the reason behind creating this (and `ClimbingNode`)?

You're not adding any new logic or overriding stuff here